MANIFEST.in

```
@@ -0,0 +1 @@
+graft mechanisms
```

```suggestion
include mechanisms/*.cti
```
(`graft` has the potential to pick up unwanted files, such as backups)

---

bb15df6

---

This change seems to break the examples.

---

No it doesn't.  09f5666